corehq/apps/app_manager/templates/app_manager/partials/releases_deploy_modal.html

```diff
@@ -184,6 +184,11 @@ <h4 class="modal-title">{% trans "Deploying version" %} <span data-bind="text: v
                                     <a target="_blank" href="https://help.commcarehq.org/display/commcarepublic/Troubleshooting">{% trans "Troubleshooting Phone Problems" %}</a>.
                                 </li>
                             </ol>
+                            <p data-bind="if: !reveal_java_download()">
+                                {% trans "This version is not compatible with Java Phones. Please create a new build with latest J2ME supported version, "%}
+                                {{ lastest_j2me_enabled_build }}
+                                {% trans "to install on Java Phone." %}
```

Comment:
This should use the [blocktrans/with](https://docs.djangoproject.com/es/1.9/topics/i18n/translation/#std:templatetag-blocktrans) pattern in case other languages don't structure the sentence in the same way.